File: library/src/main/java/com/alamkanak/weekview/EmptyViewLongPressListener.java
package com.alamkanak.weekview;

import android.support.annotation.NonNull;

import java.util.Calendar;

public interface EmptyViewLongPressListener {

    /**
     * Similar to {@link EmptyViewClickListener} but with long press.
     *
     * @param time: {@link Calendar} object set with the date and time of the long pressed position on the view.
     */
    void onEmptyViewLongPress(@NonNull Calendar time);

}

File: library/src/main/java/com/alamkanak/weekview/EmptyViewLongPressListener.kt
package com.alamkanak.weekview

import java.util.*

interface EmptyViewLongPressListener {

    /**
     * Called when an empty area of [WeekView] is long-clicked.
     *
     * @param time A [Calendar] with the date and time of the clicked position
     */
    fun onEmptyViewLongPress(time: Calendar)

}
